### PR TITLE
Use Segment to record install statistics

### PR DESCRIPTION
**Pull request by asiL-tcefreP**
# Description
Add install statistics logic in .sh file. Modify user id to computer-specific identifier in ssa.go

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ ] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: install_with_binary.sh [hunk 1]
```diff
@@ -58,9 +58,36 @@ if [[ $? -ne 0 ]]; then
     echo "wget failed to get latest binary from $url"
     exit 1; 
 fi
+
+# Step 4: Install statistics
+display_step 4 "Install statistics"
+computer_specific_identifier=$(ifconfig 2>/dev/null | grep -o -E '([0-9a-fA-F]{2}:){5}([0-9a-fA-F]{2})' | head -n 1)
```
Comment (AndrewHess) [inline install_with_binary.sh:64]: Can you add a comment saying what this is getting?

**Review by AndrewHess — COMMENTED** [review]
I think we don't want the Basic auth token in version control, but I'm not sure how else we would make this work. Any thoughts on this @sunitakawane or @nkunal?

Also it looks like your logic is the same for each of the install scripts, so you should probably have this in a separate script that you call from each of the install scripts, so that if we want to change it later we'd just have to make one change instead of three identical changes. However, if you expect that we'll need different logic for the different installation methods you can keep them separate like you have them

**Comment by asiL-tcefreP** [comment]
Yes, their logic is the same. The reason I didn't separate this code into a standalone file initially was that I thought users would only download a single .sh file instead of two. If this doesn't have a significant impact, I will extract the logic of the code into a separate .sh file.

**Comment by AndrewHess** [comment]
> Yes, their logic is the same. The reason I didn't separate this code into a standalone file initially was that I thought users would only download a single .sh file instead of two. If this doesn't have a significant impact, I will extract the logic of the code into a separate .sh file.

Oh, never mind. Yes you should keep it how you have it so they only have to download one file

**Review by AndrewHess — APPROVED** [review]

**Pull request merged**